bin/genesis

```diff
@@ -5807,20 +5957,22 @@ sub {
 	my $temp = tempdir(CLEANUP => 1);
 	my $stem = "$options{name}-$options{version}";
 	mkdir_or_fail "$temp/$stem";
-	qx(cp -aH $dir/* $temp/$stem);
-
-	qx(rm -rf $temp/$stem/.git $temp/$stem/.gitignore $temp/$stem/*.tar.gz $temp/$stem/ci);
-	qx(echo "version: $options{version}" | spruce merge $dir/kit.yml - > $temp/$stem/kit.yml);
+	Genesis::ShellExec::check('cp -aH "${1}"/* "${2}/${3}"', $dir, $temp, $stem);
+	Genesis::ShellExec::check('rm -rf "${1}/${2}/"{.git,.gitignore,*.tar.gz,ci}', $temp, $stem);
+	Genesis::ShellExec::check(
+		'echo "version: ${1}" | spruce merge "${2}/kit.yml" - > "${3}/${4}/kit.yml"',
+		$options{version}, $dir, $temp, $stem
+	);
 
 	if (!$options{force}) {
-		my $changes = qx(git status --porcelain);
+		my $changes = Genesis::ShellExec::check('git status --porcelain');
 		if ($? == 0 && $changes) {
 			error "\n#R{ERROR}: Unstaged / uncommited changes found in working directory.\n".
 			      "Please either #C{stash} or #C{commit} those changes before compiling your kit.\n";
 			exit 1;
 		}
 	}
-	qx(tar -czf $stem.tar.gz -C $temp $stem/);
+	Genesis::ShellExec::check('tar -czf "${2}.tar.gz" -C "$1" "$2/"', $temp, $stem);
```

Comment:
It seems like all these check() calls should be reacted to.  Might I interest you in `Genesis::ShellExec::Some::More::Modules::For::Fun::run()`?